packages/property-grid/src/components/MultiElementPropertyGrid.tsx

```diff
@@ -170,31 +170,34 @@ export const MultiElementPropertyGrid = (props: Partial<PropertyGridProps>) => {
 
   return (
     <div className="property-grid-react-transition-container">
-      <Transition
-        items={content as number}
-        config={{ duration: 250, easing: (t: number) => t * t }}
-        from={{
-          transform: animationForward
-            ? "translate(100%,0)"
-            : "translate(-100%,0)",
-        }}
-        enter={{ transform: "translate(0,0)" }}
-        leave={{
-          transform: !animationForward
-            ? "translate(100%,0)"
-            : "translate(-100%,0)",
-        }}
-      >
-        {(index) => (style) =>
-          (
-            <animated.div
-              className="property-grid-react-animated-tab"
-              style={style}
-            >
-              {items[index]}
-            </animated.div>
-          )}
-      </Transition>
+      <div className="property-grid-react-transition-container-inner">
+        <Transition
+          items={content as number}
+          config={{ duration: 250, easing: (t: number) => t * t }}
+          from={{
+            transform: animationForward
+              ? "translate(100%,0)"
+              : "translate(-100%,0)",
+          }}
+          enter={{ transform: "translate(0,0)" }}
+          leave={{
+            transform: !animationForward
+              ? "translate(100%,0)"
+              : "translate(-100%,0)",
+          }}
+        >
+          {/* eslint-disable-next-line react/display-name */}
```

Comment:
should probably disable this globally. Doesnt make a ton of sense for functional components IMO

Reply:
Idk i think this can be helpful even for functional components. If components don't have display names, theyll show up in dev tools as anonymous. It can happen if you do default unnamed exports (im not a fan of this to begin with, but have seen this in some of our codebases)

Reply:
fair enough. Should we add a name to this component then instead of disabling the rule?

Reply:
probably should, i just couldnt figure out how to do so with this nested component.
im hoping to nuke this transition wraper tomorrow if i get a chance, i dont love that we pull in an entire library to do what should be a simple animation. also causes us to add a peer dep in the viewer 9react-spring)